api/v1alpha1/cyndipipeline_types.go

```diff
@@ -56,6 +56,10 @@ type CyndiPipelineSpec struct {
 	// +optional
 	// +kubebuilder:validation:MinLength:=1
 	InventoryDbSecret *string `json:"inventoryDbSecret,omitempty"`
+
+	// +optional
+	// +kubebuilder:validation:MinLength:=1
+	DBTableIndexSQL string `json:"dbTableIndexSQL,omitempty"`
```

Comment:
Doesn't this grant full access to the database?

Reply:
I'm concerned about this too - what guards are in place?

Reply:
This is similar to how the existing table init SQL is defined. It can be overridden via a [configmap entry](https://github.com/RedHatInsights/cyndi-operator/blob/1d64e9bcf9cb0a2fabbd50806cff9f8db042b734/controllers/config/config.go#L84). Instead of a single configmap entry to override the sql for all apps, this allows individual apps to override it. It must be defined/approved via an app-interface MR.

The SQL is executed in the operator using the `cyndi` user which only has access to the `inventory` schema.